ts/TexturePreloader.ts

```diff
@@ -7,12 +7,19 @@ class CardArtPreloader extends Stream.Writable {
 	protected cardArtQueue = ['GAME_005'];
 	protected images = [];
 	private working = 0;
-	protected assetQueue = ['cardback', 'hero_frame', 'hero_power', 'inhand_minion', 'inhand_spell', 'inhand_weapon',
+	protected assetQueue = ['cardback', 'hero_frame', 'hero_power',
+							'inhand_minion_druid', 'inhand_minion_hunter', 'inhand_minion_mage',
+							'inhand_minion_paladin', 'inhand_minion_priest', 'inhand_minion_rogue',
+							'inhand_minion_shaman', 'inhand_minion_warlock', 'inhand_minion_warrior',
+							'inhand_minion_neutral', 'inhand_spell_druid', 'inhand_spell_hunter',
+							'inhand_spell_mage', 'inhand_spell_paladin', 'inhand_spell_priest', 'inhand_spell_rogue',
+							'inhand_spell_shaman', 'inhand_spell_warlock', 'inhand_spell_warrior', 'inhand_spell_neutral',
```

Comment:
You're probably right, no sense preloading all of them. Maybe just strip it back to the neutrals.